src/BBT.StrategyPattern/BBT.StrategyPattern.csproj

```diff
@@ -35,5 +35,9 @@
       <_Parameter1>$(MSBuildProjectName).Tests</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
+  
+  <ItemGroup>
+    <Folder Include="Strategy\" />
```

Comment:
Remove this and keep all files in the root. For the consumer it's much more convenient to have all classes in a single namespace instead of requiring to import 3 namespaces.